packages/tasks/src/tasks/image-to-3d/about.md

```diff
@@ -0,0 +1,50 @@
+## Use Cases
+
+Image-to-3D models can be used in a wide variety applications that require 3D, such as games, animation, design, architecture, engineering, marketing, and more.
+
+### Meshes
+
+Meshes are the standard representation of 3D in industry.
+
+Meshes can be in `.obj`, `.glb`, `.stl`, or `.gltf` format. Other formats are allowed, but won't be rendered in the gradio [Model3D](https://www.gradio.app/docs/gradio/model3d) component.
+
+### Splats
+
+[Gaussian Splatting](https://huggingface.co/blog/gaussian-splatting) is a differentiable rasterization technique that represents 3D as fuzzy points.
+
+Splats can be in `.ply` or `.splat` format. They can be rendered in the gradio [Model3D](https://www.gradio.app/docs/gradio/model3d) component using the [gsplat.js](https://github.com/huggingface/gsplat.js) library.
```

Comment:
I feel like this is a bit drowning the use case itself in general sense, you can put these details somewhere below instead